docs/Overview.md

```diff
@@ -0,0 +1,291 @@
+# @remoteoss/remote-flows
+
+## Overview
+
+Welcome to the `@remoteoss/remote-flows` package, a React library that provides components for Remote's embbeded solution.
+
+## Installation
+
+```sh
+npm install @remoteoss/remote-flows
+```
+
+## Getting Started
+
+After installation, import the main CSS file in your application:
+
+```css
+@import '@remoteoss/remote-flows/index.css';
+```
+
+### Basic Setup
+
+Here's a simple example of how to set up the library:
+
+```tsx
```

Comment:
> Here's a simple example of how to set up the library:

I don't think the example bellow is the most simple 😅 I think we should have 2/3 examples, starting from the most basic to most complex (more functionality). I would write an example for:
- basic scenario rendering `<CostCalculator />`
- follow up scenario, showcasing how `CostCalculatorResults` can be used
- final example, showcasing how to export results in pdf